src/test/scala/mesosphere/marathon/api/akkahttp/v2/AppsControllerTest.scala

```diff
@@ -56,7 +57,10 @@ class AppsControllerTest extends UnitTest with GroupCreation with ScalatestRoute
 
   case class Fixture(
       clock: SettableClock = new SettableClock(),
-      auth: TestAuthFixture = new TestAuthFixture(),
+      auth: TestAuthFixture = new TestAuthFixture {
+        override val UnauthorizedStatus = 403
```

Comment:
Still wondering if we broke the API. Yes, we might have been wrong originally but we cannot fix it withing v2.